smdebug/core/config_constants.py

```diff
@@ -41,3 +41,5 @@
 
 CALLABLE_CACHE_ENV_VAR = "SMDEBUG_KERAS_CALLABLE_CACHE_TYPE"
 DEFAULT_CALLABLE_CACHE = "CACHE_PER_MODE"
+
+DEFAULT_SAVED_COLLECTIONS = ["losses", "metrics", "sm_metrics"]
```

Comment:
AFAIK metrics are relevant only to TF. will this fail for MX and PT?

Reply:
Okay, didn't know that.
And the tests failed for MxNet and Pytorch